spec/javascripts/Policies/components/CustomPolicy.spec.jsx

```diff
@@ -76,6 +76,21 @@ describe('CustomPolicyForm', () => {
     expect(wrapper.find('input[name="schema"]').prop('value')).toBe(JSON.stringify(POLICY_TEMPLATE.schema))
     expect(wrapper.find('input[name="_method"]').exists()).toBe(false)
   })
+
+  // TODO: remove `skip` when this is merged: https://github.com/airbnb/enzyme/pull/2008
+  it.skip('should change submit method when clicking Delete Policy', () => {
```

Comment:
Following up on previous comments, this would not make sense if using a hook is avoided.

To make this spec more robust (or perhaps an additional spec), it could instead check the form submitted instead? Adding a spy on `onSubmit` and asserting the proper action is being submitted.

I mean, the only "gap" I see is we are not testing the actual submit.

Reply:
The submit of the form is not needed since it's a traditional form submit. What it's needed is, in fact, the modification of that hidden input. I've explained above the importance of it with the Rails MGMT

Reply:
OK, I see your point here. However we should verify that this value is set **before** the form is submitted, otherwise it's a false positive. Perhaps asserting `onSubmit` hasn't been called before asserting the value of `_method`?

Reply:
I could do in the next iteration when enzyme merges that PR a programmatically submit and assert that the field has been modified.

Reply:
Last comment I promise: could you add todo for this so we don't forget?
```
it.todo('should blah blah')
```

Reply:
Would change the test description and add an additional `TODO` note about it

Reply:
Ah yes, of course